jobs/fullstack_developer_at_TOP-IX.html

```diff
@@ -0,0 +1,45 @@
+---
+title: Full-Stack Developer
+company: TOP-IX Consortium
+url: http://www.top-ix.org
+location: Turin, Italy
+contract: permanent
+contact: staff@top-ix.org
+created: !!timpestamp '2018–02-01’
```

Comment:
Typo in `timestamp` here

Reply:
And also the closing quote is a smartquote - I think that's the error that Travis CI is complaining about.